app-backend/build.sbt

```diff
@@ -117,3 +118,16 @@ lazy val database = Project("database", file("database"))
   .settings({
    libraryDependencies ++= slickDependencies ++ dbDependencies ++ loggingDependencies ++ Seq(Dependencies.akkaHttpExtensions)
   })
+
+lazy val tiler = Project("tileserver", file("tileserver"))
+  .settings(commonSettings:_*)
+  .settings(resolvers += Resolver.bintrayRepo("azavea", "geotrellis"))
+  .settings({
+    libraryDependencies ++= loggingDependencies ++ Seq(
+      "org.apache.spark" %% "spark-core" % Version.spark,
```

Comment:
Also, we have to be a little careful here - spark can't be marked as provided since we will be running the tile server using `java -jar rf-tileserver.jar` when this is deployed.